ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.18__arm64-nightly.yaml

```diff
@@ -329,7 +329,7 @@ tests:
       CONTROL_PLANE_INSTANCE_TYPE: m6g.xlarge
       OCP_ARCH: arm64
       TEST_SCENARIOS: API_Server
-      TEST_TIMEOUT: "600"
+      TEST_TIMEOUT: "120"
```

Comment:
@wangke19 Why we changing from 600 to 120?

Reply:
600 minutes, it's too longer. I tested [one 4.18 job](https://qe-private-deck-ci.apps.ci.l2s4.p1.openshiftapps.com/view/gs/qe-private-deck/pr-logs/pull/openshift_release/60157/rehearse-60157-periodic-ci-openshift-openshift-tests-private-release-4.18-arm64-nightly-aws-ipi-f9-longduration-apiserver/1872158409436958720) with new timeout value, it got passed.